data/_models/shared/Collections.csv

```diff
@@ -117,7 +118,8 @@ Collections,,lag time,int,,,,,,,,,"How many days is the lag time after refresh b
 Collections,,preservation,bool,,,,,,,,,Are records preserved in the data bank indefinitely?,"DataCatalogueFlat,EMA","['Databank', 'Data source'].includes(collectionType?.name)"
 Collections,,preservation duration,int,,,,,,,,,"If no to the above, for how long (in years) are records preserved in the data bank?","DataCatalogueFlat,EMA","['Databank', 'Data source'].includes(collectionType?.name)"
 Collections,,refresh period,ontology_array,,,CatalogueOntologies,Refresh periods,,,,,"If data are refreshed on fixed dates (e.g., every June and December), when are the refreshes scheduled? Select all that apply from the following:","DataCatalogueFlat,EMA","['Databank', 'Data source'].includes(collectionType?.name)"
-Collections,,date last refresh,date,,,,,,,,,Date of last update/refresh,"DataCatalogueFlat,EMA","['Databank', 'Data source'].includes(collectionType?.name)"
+Collections,,date issued,date,,,,,,,,dct:issued,Date of formal issuance (e.g. publication) of the collection,"DataCatalogueFlat,EMA","['Databank', 'Data source'].includes(collectionType?.name)"
```

Comment:
ik stel voor dat we dit niet doen maar daarvoor in de plaats mg_insertedOn en mg_updatedOn gebruiken. Deze kolommen kunnen dan weg (mits we mogelijk maken als admin die velden bij te werken)

Als je deze leeg maakt en andere comments addreseert kan deze gemerged.

Reply:
Bovendien, door de manier waarop we nu data migreren, worden `mg_insertedOn` en `mg_updatedOn` bij iedere migratie gezet op het moment van de migratie. Die wijkt af van het moment van inhoudelijke update en al helemaal van het moment van eerste insertion

Reply:
Er is wel een feature request om mg_insertedOn en mg_updatedOn te kunnen exporteren/importeren, toch? EMA heeft dit bijv. ook nodig.

Reply:
als jullie dit ook zo zien kunnen we PR maken die dit standaard toevoegt

Reply:
created #3890 for this